src/schematics/ng-add/index.ts

```diff
@@ -60,21 +61,29 @@ function addLoadersToPackageJson() {
  */
 function addPugRules() {
   return (tree: Tree) => {
-    const configFile = tree.read(TARGET_CONFIG_PATH);
-    if (!configFile)
+    const commonConfigFile = tree.read(TARGET_COMMON_CONFIG_PATH);
+    const typescriptConfigFile = tree.read(TARGET_TYPESCRIPT_CONFIG_PATH);
+      
+    if (!commonConfigFile || !typescriptConfigFile)
```

Comment:
The tests are failing because of this line. The config file in `TARGET_COMMON_CONFIG_PATH` is created for the test suite at [src/utils/testing.ts#createCommonWebpackConfig](https://github.com/danguilherme/ng-cli-pug-loader/blob/v0.1.7/src/utils/testing.ts#L33-L42). You would need to create a test file for `TARGET_TYPESCRIPT_CONFIG_PATH` too.

Reply:
Thanks for explaining it, I've never done unit testing before.  I fixed these and added one for Angular 8 behavior check.